03-task-manager/starter/middleware/error-handler.js

```diff
@@ -0,0 +1,9 @@
+const { CustomAPIError } = require('../errors/custom-error')
```

Comment:
not blocking for assignment submission but you'll want to import these into your app.js and `use` them to get full functionality